scripts/run_gori_lstm_v2.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-#SBATCH -J conv2d_test
+#SBATCH -J lstmv_v2_test
 #SBATCH -t 02:00:00
 #SBATCH -C gpu
-#SBATCH --gres=gpu:8
+#SBATCH --gres=gpu:1
```

Comment:
we need 8 GPUs to make sure other people won't run jobs simultaneously, especially for profiling